distributed/client.py

```diff
@@ -3872,7 +3930,8 @@ async def _get_versions(self, check=False, packages=[]):
             scheduler = await self.scheduler.versions()  # this raises
 
         workers = await self.scheduler.broadcast(
-            msg={"op": "versions", "packages": packages}
+            msg={"op": "versions", "packages": packages},
+            on_error="ignore",
```

Comment:
Sorry, one last comment after thinking about this a bit more. We often ask users to inspect the output of `Client.get_versions` when debugging package version mismatches. With `on_error="ignore"` here we could run into cases where there's a problematic version mismatch on an unresponsive worker which will be silently ignored (and hard to debug). What happens today? My guess is a connection error is raised

Thoughts on emitting a warning instead of silently ignoring?

Reply:
Now logging an error in the scheduler log